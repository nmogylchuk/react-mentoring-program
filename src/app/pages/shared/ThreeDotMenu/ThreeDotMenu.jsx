import React from 'react';
import './ThreeDotMenu.scss';

function ThreeDotMenu() {

    return (
        <div className="three-dots-menu"></div>
    )
}

export default ThreeDotMenu;
