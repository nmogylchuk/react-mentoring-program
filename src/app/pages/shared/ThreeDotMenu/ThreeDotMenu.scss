@import './../../../assets/styles/variables';

.three-dots-menu {
  position: absolute;
  right: 0;
  padding: 3px 4px 0 9px;
  margin: 5px;
  background-color: $black;
  border: 1px solid $white;
  border-radius: 20px;
  opacity: .7;

  &:after {
    content: '\2807';
    font-size: 2rem;
  }
}
