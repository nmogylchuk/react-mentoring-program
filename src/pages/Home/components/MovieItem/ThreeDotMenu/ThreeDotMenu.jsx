import React from 'react';
import 'pages/Home/components/MovieItem/ThreeDotMenu/ThreeDotMenu.scss';

const ThreeDotMenu = (props) => {
    const {showMenu} = props;
    return (
        <div className="three-dots-menu" onClick={showMenu}></div>
    );
};

export default ThreeDotMenu;
