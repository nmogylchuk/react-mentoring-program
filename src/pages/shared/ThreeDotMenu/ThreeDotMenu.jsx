import React from 'react';
import 'pages/shared/ThreeDotMenu/ThreeDotMenu.scss';

function ThreeDotMenu() {

    return (
        <div className="three-dots-menu"></div>
    )
}

export default ThreeDotMenu;
