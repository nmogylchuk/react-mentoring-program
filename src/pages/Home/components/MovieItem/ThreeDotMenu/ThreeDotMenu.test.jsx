import React from 'react';
import rerender from 'react-test-renderer';
import ThreeDotMenu from "./ThreeDotMenu";

describe('when Three Dot Menu', () => {

    test('then snapshot created', () => {
        const threeDotMenuComponent = rerender.create(<ThreeDotMenu showMenu={true}/>);
        const tree = threeDotMenuComponent.toJSON();
        expect(tree).toMatchSnapshot();
    });

});