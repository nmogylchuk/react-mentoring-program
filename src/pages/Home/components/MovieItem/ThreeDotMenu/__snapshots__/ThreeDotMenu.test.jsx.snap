// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when Three Dot Menu then snapshot created 1`] = `
<div
  className="three-dots-menu"
  onClick={true}
/>
`;
